spec/support/api_helpers.rb

```diff
@@ -204,6 +207,22 @@ def response_body
     last_response.body
   end
 
+  def search_api_get(*args, **keywords)
+    api_request(:get, *args, **{ version: 'search/v1' }.merge(keywords))
+  end
+
+  def search_api_post(*args, **keywords)
+    api_request(:post, *args, **{ version: 'search/v1' }.merge(keywords))
+  end
+
+  def search_api_delete(*args, **keywords)
+    api_request(:delete, *args, **{ version: 'search/v1' }.merge(keywords))
+  end
+
+  def search_api_put(*args, **keywords)
+    api_request(:put, *args, **{ version: 'search/v1' }.merge(keywords))
+  end
```

Comment:
why'd you make these helpers for post/delete/put?

Reply:
Oh, I just copied and pasted.  Can remove for now.